syncd/scripts/syncd_init_common.sh

```diff
@@ -158,6 +158,15 @@ config_syncd_vs()
     CMD_ARGS+=" -p $HWSKU_DIR/sai.profile"
 }
 
+config_syncd_innovium()
+{
+    CMD_ARGS+=" -p $HWSKU_DIR/sai.profile"
+    ulimit -s 65536
```

Comment:
why do need this?

Reply:
We see stack usage going beyond 8K in certain scenario causing syncd crash.